lms/djangoapps/country_regions_api/views.py

```diff
@@ -0,0 +1,49 @@
+from django_countries import countries
+from pymongo import MongoClient
+from pymongo.mongo_client import ConnectionFailure
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from util.json_request import JsonResponse
+from .helpers import PyMongoJSONEncoder
+import json
+
+
+class CountryRegionsView(APIView):
+    def __mongo_connect(self):
+        """
+        local mongo connect and return collection
+        country subdivision
+        :return: regions_collection
+        """
+        try:
+            client = MongoClient('mongodb://localhost:27017/')
+        except ConnectionFailure:
+            return None
+        db = client['edxapp']
+        regions_collection = db['regions']
+        return regions_collection
+
+    def _alpha2_to_alpha3(self, alpha2_code):
+        """
+        Adapter for translate country Alpha2 code to Alpha 3
+        Example: UA -> UKR, US -> USA
+        :param alpha2_code
+        :type alpha2_code: str
+        :return alpha3_code
+        :type alpha3_code: str(unicode)
+        """
+        return countries.alpha3(alpha2_code)
+
+    def get(self, request):
+        country_alpha3_code = self._alpha2_to_alpha3(request.GET.get('country_alpha2'))
+        regions_collection = self.__mongo_connect()
+        if regions_collection:
+            country_regions = regions_collection.find_one({"country_name": country_alpha3_code})
```

Comment:
`find_one` returns `None` if there is no results. Please provide avoiding error from the next line

Reply:
On the next line is being implemented check for None value